core/domain/exp_jobs_one_off.py

```diff
@@ -685,3 +689,90 @@ def reduce(status, values):
+
+
+class ValidationOfImagesOnGCS(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for checking that all the images in the GAE are there in
+        the GCS or not.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [file_models.FileModel]
+
+    @staticmethod
+    def map(file_model):
+        # This job is allowed to run only in Production environment since it
+        # uses GcsFileSystem which can't be used in Development environment.
+        if not feconf.DEV_MODE:
+            instance_id = file_model.id
+            filetype = instance_id[instance_id.rfind('.') + 1:]
+            # To separate the image entries from the audio entries we get from
+            # the FileSnapshotContentModel.
+            if filetype in ALLOWED_IMAGE_EXTENSIONS:
+                pattern = re.compile(
+                    r'^/([^/]+)/assets/(([^/]+)\.(' + '|'.join(
+                        ALLOWED_IMAGE_EXTENSIONS) + '))$')
+                catched_groups = pattern.match(instance_id)
+                if not catched_groups:
+                    yield (WRONG_INSTANCE_ID, instance_id)
+                else:
+                    filename = catched_groups.group(2)
+                    exploration_id = catched_groups.group(1)
+                    fs = fs_domain.AbstractFileSystem(
+                        fs_domain.GcsFileSystem(exploration_id))
+
+                    if not fs.isfile('image/%s' % filename):
+                        yield (FILE_IS_NOT_IN_GCS, file_model.id)
+
+
+    @staticmethod
+    def reduce(status, values):
+        yield (status, values)
+
+
+class DeleteImagesFromGAEJob(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for deleting the images in the exploration
+        from the GAE.
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [file_models.FileModel]
+
+    @staticmethod
+    def map(file_model):
+        instance_id = file_model.id
+        filetype = instance_id[instance_id.rfind('.') + 1:]
+        # To separate the image entries from the audio entries we get from
+        # the FileSnapshotContentModel.
+        if filetype in ALLOWED_IMAGE_EXTENSIONS:
+            pattern = re.compile(
+                r'^/([^/]+)/assets/(([^/]+)\.(' + '|'.join(
+                    ALLOWED_IMAGE_EXTENSIONS) + '))$')
+            catched_groups = pattern.match(instance_id)
+            if not catched_groups:
+                yield (WRONG_INSTANCE_ID, instance_id)
+            else:
+                filename = catched_groups.group(2)
+                filepath = 'assets/' + filename
+                exploration_id = catched_groups.group(1)
+                filemetadata_model = (
+                    file_models.FileMetadataModel.get_model(
+                        exploration_id, filepath, False))
+
+                file_model.delete(
+                    'ADMIN',
+                    'Deleting file_model for image from GAE',
+                    True)
```

Comment:
If there's a default value for an arg then the arg name should be given in the caller.

@apb7 I thought there was supposed to be a lint check for this? /cc @kevinlee12

Reply:
Done!